.github/workflows/ci-testing-pull-request.yml

```diff
@@ -59,7 +59,7 @@ jobs:
       - name: check api-server backwards compatibility
         run: |
           ./scripts/openapi-diff.bash breaking --fail-on ERR\
-            https://raw.githubusercontent.com/${{ github.event.pull_request.base.repo.full_name }}/${{ github.event.after }}/services/api-server/openapi.json \
+            https://raw.githubusercontent.com/${{ github.event.pull_request.base.repo.full_name }}/refs/heads/${{ github.base_ref }}/services/api-server/openapi.json \
```

Comment:
I am confused. I would have swore I already did this sometime ago ... ! :-(

Reply:
https://github.com/itisfoundation/osparc-simcore/commit/0eca9c2a63845428781c7e70f1c7ff7d1fc7b276

AH, it was only in one fo the jobs!